os/kernel/src/drivers/pci/pci.c

```diff
@@ -4,33 +4,34 @@
 #include <stdlib.h>
 #include <ktime.h>
```

Comment:
Why do you need a ktime?

Reply:
huh, it must've been leftover, removing this